json-lint/schemas/inventory/adservers/disqus.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$id": "#disqus",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Disqus adserver for TOUCH only",
+    "description": "Disqus ad server for TOUCH only",
+    "additionalProperties": false,
+    "required": [
+        "type",
+        "shortName",
+        "forum",
+        "category",
+        "tid",
+        "position",
+        "json"
+    ],
+    "defaultSnippets": [
+        {
+            "label": "Disqus template",
+            "description": "contains required keys",
+            "body": {
+                "type": "disqus",
+                "id": "$1"
```

Comment:
This can have all the others required: 
`` "shortName",
        "forum",
        "category",
        "tid",
        "position"``